magefile.go

```diff
@@ -32,6 +32,8 @@ import (
 
 	devtools "github.com/elastic/beats/v7/dev-tools/mage"
 	"github.com/elastic/beats/v7/dev-tools/mage/gotool"
+        // mage:import
```

Comment:
`mage:import` is going to import all the targets defined in `unittest`, so there is no need to re-define `UnitTest` target if it only calls to `unittest.UnitTest`.